containers/Makefile

```diff
@@ -164,28 +160,27 @@ build/%: %
 
 # Templates
 build/%: %.j2 build/config-with-versions.yml ../tools/render-jinja-template.sh
-	../tools/render-jinja-template.sh         \
-		build/config-with-versions.yml        \
-		"$<"                                  \
+	../tools/render-jinja-template.sh\
+		build/config-with-versions.yml \
+		"$<"                           \
 		"$@"
 
 build/config-with-versions.yml: versions.yml config.yml ../tools/merge-yaml.py
-	../tools/merge-yaml.py                     \
-		"$<"                                   \
-		"config.yml"                           \
+	../tools/merge-yaml.py\
+		"$<"                \
+		"config.yml"        \
 		--output "$@"
 
-
 clean:
 	rm -rf build/
 
-
-.PHONY:                                 \
-    all                                 \
-    nginx-proxy   nginx-proxy-clean     \
-    nginx-storage nginx-storage-clean   \
-    postgresql    postgresql-clean      \
-    concent-api   concent-api-clean     \
-    verifier      verifier-clean        \
-    docker-clean                        \
-    clean
+.PHONY:                            \
+	all                              \
+	nginx-proxy   nginx-proxy-clean  \
+	nginx-storage nginx-storage-clean\
+	postgresql    postgresql-clean   \
+	concent-api   concent-api-clean  \
+	verifier      verifier-clean     \
+	docker-clean                     \
+	clean                            \
+	%-rmi
```

Comment:
Are you sure this works? I think that pattern rules will only work if `%` is in the target as well. But I may be wrong. Please check.

Reply:
That is correct,  I did some tests and .PHONY targets must be declared explicitly.